packages/api/src/entity/subscription.ts

```diff
@@ -70,4 +70,10 @@ export class Subscription {
 
   @Column('timestamp', { nullable: true })
   scheduledAt?: Date | null
+
+  @Column('boolean')
+  isPublic?: boolean | null
+
+  @Column('boolean')
+  isFetchingContent?: boolean | null
```

Comment:
I wonder if we should use a state enum instead of a bool

Reply:
Ahh the name is misleading. It's actually a flag to indicate if we want to fetch content

Reply:
@jacksonh maybe change this one to `shouldFetchContent`?